src/main/resources/data/create/alloy_forge_fuels/fuels.json

```diff
@@ -0,0 +1,12 @@
+{
+  "fuels": [
+    {
+      "item": "create:blaze_cake",
+      "fuel": 6000
+    }, 
+    {
+      "item": "create:creative_blaze_cake",
+      "fuel": 2147483647
```

Comment:
Having the fuel value set this high will cause an integer overflow, causing the fuel of any Forge Controller to become negative

Reply:
This value is the integer limit. 1 higher than this will overflow. This value is fine. I checked alloy forgery and it should handle values greater than shorts fine.

Reply:
okay, so this will overflow due to how AF handles cap checking. Do `192000` instead to match the limit.